cumulus_library/databases/duckdb.py

```diff
@@ -207,6 +182,28 @@ def operational_errors(self) -> tuple[type[Exception], ...]:
             duckdb.BinderException,
         )
 
+    def export_table_as_parquet(
+        self, table_name: str, table_type: str, location: pathlib.Path, *args, **kwargs
+    ) -> str | None:
+        parquet_path = location / f"{table_name}.{table_type}.parquet"
+        parquet_path.parent.mkdir(exist_ok=True, parents=True)
```

Comment:
The caller already did this yeah? - Maybe add a comment in the docstring for the abstract method about what the assumption about the dir is.